doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Sonata project contribution
+
+Thanks for you interest onto Sonata projects!
+
+## Summary
+
+* [Issues]()
+* [Pull Requests](#pull-requests)
+* [Label rules]()
+
+## Pull Requests
+
+All the sonata team will be glad to review your code changes propositions! :smile:
+
+But please, read the following before.
+
+### Coding style
+
+Each project follows [PSR-1](http://www.php-fig.org/psr/psr-1/), [PSR-2](http://www.php-fig.org/psr/psr-2/)
+and [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) for coding style,
+[PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading.
+
+Please [install PHP Coding Standard Fixer](http://cs.sensiolabs.org/#installation)
+and run this command before committing your modifications:
+
+```bash
+php-cs-fixer fix --verbose
+```
+
+### Writing a Pull Requests
```

Comment:
@greg0ire I was not specifically talking about *minor* changes. But more about yet undiscovered issues you trigger when doing a modification, and that prevent you from going any further (e.g. it makes the tests go red, or raise an Exception that shouldn't happen).
I have an example (sorry for the long comment) that occured while working on PR:
I needed a way to pass data from the FieldDescription object to the js layer (no [writing js in templates](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/master/Resources/views/CRUD/edit_orm_many_association_script.html.twig#L33) allowed).
So I opted for passing this data as a JSON string into an HTML data attribute, standard stuff.
This triggered the case explained in [this issue](https://github.com/sonata-project/SonataAdminBundle/issues/3646), which i'll try to sum up quickly:
When you put a json string in a twig template, it gets escaped, which is good. But now if you want to unit test the template's output, you have to either:
* encode the _expected_ value by hand? no way!
* call twig_escape_filter() on the expected value inside a `@dataProvider`? nope.
* write the expected value in twig and compile it on the fly ? slow and meh and wtf?.
* take a step back and tell yourself: "Maybe the tests are testing the wrong thing? They compare strings where they should compare the DOM... so let's write an [assertion to do that](https://github.com/ju1ius/SonataAdminBundle/blob/js-refactoring/Tests/HTMLAssert.php#L48), and [use it](https://github.com/ju1ius/SonataAdminBundle/blob/js-refactoring/Tests/Twig/Extension/SonataAdminExtensionTest.php#L290)"

I hope you see were this is going: if I have to make these changes in another branch, my main PR branch now contains tests that cannot possibly pass because the changes needed to make them green are in another branch !  :sob: